pkg/brctl/brctl.go

```diff
@@ -276,29 +276,41 @@ func showBridge(name string, out io.Writer) error {
 	return nil
 }
 
-// Showmacs shows a list of learned MAC addresses for this bridge.
-// The mac addresses are stored in the first 6 bytes of /sys/class/net/<name>/brforward,
-// The following format applies:
-// 00-05: MAC address
-// 06-08: port number
-// 09-10: is_local
-// 11-15: timeval (ignored for now)
-func Showmacs(bridge string, out io.Writer) error {
-	// parse sysf into 0x10 byte chunks
+// ShowMACs shows a list of learned MAC addresses for this bridge.
+// The following byte format applies according to the kernel source [1]
+// 0-5:   MAC address
+// 6:     port number
+// 7:     is_local
+// 8-11:  ageing timer
+// 12-15: unused in this context
+//
+// [1] https://github.com/torvalds/linux/blob/master/include/uapi/linux/if_bridge.h#L93
+func ShowMACs(bridge string, out io.Writer) error {
```

Comment:
Does it make sense to maybe add a unit test for an exemplary run for a known good test file to ensure functionality? The `io.Writer` should make mocking easier.

Reply:
Definitely. As I mentioned above. I want to file another PR incl. integration tests and general test improvements